watertap/unit_models/ion_exchange_0D.py

```diff
@@ -387,12 +525,12 @@ def build(self):
         )
 
         self.bed_depth = Var(
-            initialize=1, bounds=(0, 2), units=pyunits.m, doc="Bed depth"  # EPA-WBS
+            initialize=1, bounds=(0, 3.0), units=pyunits.m, doc="Bed depth"  # EPA-WBS
         )
 
         self.bed_porosity = Var(
             initialize=0.5,
-            bounds=(0.45, 0.65),
+            bounds=(0.3, 0.8),
```

Comment:
Did you observe effects on model stability without bounds? Do problems occur if you expand upper bound to be very close to 1? I understand it may not be practical for use cases but just curious.

Reply:
I have never looked at model stability related to bed porosity. This is something I assume the user would provide, and I expanded the bounds here to be >inclusive of some values I saw while looking at PFAS-specific resins.